app/react-native/tsconfig.json

```diff
@@ -1,13 +1,12 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
-    "jsx": "react-native",
+    "jsx": "react",
```

Comment:
This is another tricky issue, but based on https://www.typescriptlang.org/docs/handbook/jsx.html#basic-usage , it seems that the difference is:

* `react` = transforms JSX syntax into `React.createElement` calls, so the code can run e.g. on the browser that doesn't understand JSX
* `react-native` = leaves JSX syntax as-is (which works when running on React Native, but browsers can't understand it)

There are multiple options and as you say, it is possible that changing this could have some negative side-effects. While I didn't identify any issues while testing, that's of course not a guarantee that we won't run into something. We could perhaps run with this and revert & research more if we notice any undesirable effects.

Reply:
Ok makes sense I guess there's no harm in changing this.